web-registry/src/components/organisms/EcocreditsTable.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { Box, styled } from '@mui/material';
+import { Box } from '@mui/material';
+import { styled } from '@mui/material';
```

Comment:
why this change?

Reply:
I was undoing a change. I forgot to check the automatic import. Thanks!